backend/clubs/views.py

```diff
@@ -2895,6 +2895,14 @@ def get(self, request):
         return Response({"count": ans})
 
     def post(self, request):
+        if settings.ZOOM_VERIFICATION_TOKEN is not None:
+            authorization = request.META["HTTP_AUTHORIZATION"]
```

Comment:
You can use a `request.META.get()` to return 403 instead of error if they do not specify the header. (`[]` errors, `.get()` returns None if not exists)